cmd/run.go

```diff
@@ -44,10 +44,14 @@ func RunCmd() *cobra.Command {
 	return cmd
 }
 
+func RunRun(ro options.RunOptions, args []string) error {
```

Comment:
Similarly, what's going on here?

Reply:
Assuming this is the same as above and for use in Scribe, just rename the existing function